### PR TITLE
Fix CircleCI build

### PR DESCRIPTION
**Pull request by Geod24**
Should be trivial.

### CODE REVIEW

**Comment by Geod24** [comment]
I cannot really test this until our website is updated. Should we revert to using github import for the moment ?

**Comment by Geod24** [comment]
@outsideris : Looks like it's [not quite working](https://circleci.com/gh/bosnet/sebak/73) yet :(

File: .circleci/config.yml [hunk 1]
```diff
@@ -4,7 +4,7 @@ jobs:
     docker:
       - image: circleci/golang:1.9
 
-    working_directory: /go/src/github.com/owlchain/sebak
+    working_directory: /go/src/github.com/bosnet/sebak
```
Comment (outsideris) [inline .circleci/config.yml]: I found the problem.

This should be changed `boscoin.io/sebak`

Comment (Geod24) [reply .circleci/config.yml]: 🤦‍♂️ 
Of course, sorry

**Pull request merged**